### DISCUSSION TITLE
Beginner level help with Podman Update

### DISCUSSION THREAD

**Discussion by iamtapu** [unifi-utilities/unifi-common#467] [Q&A]
Would appreciate some help for me to figure out how I can update Podman.
Disclaimer: I have beginner level expertise in Linux environment. 

Help needed in installing Podman Update.
**Link**: https://github.com/unifi-utilities/unifios-utilities/tree/main/podman-update
**Difficulty**: This page details might be targeted to a medium level Linux user. I am lost what I need to do. 

**Ultimate goal**: My understand is, if I can update podman then I can somehow run a docker-compose file by executing command from a some other machine and make the docker-compose run in the UDM console. I am planning on running [cloudflared](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/installation/) tunnel which can be run on a docker compose. (Please correct me if my understanding is wrong)

**What I have done so far**:
I have a UDM device. 
I am pretty sure I have completed the steps mentioned in here: https://github.com/unifi-utilities/unifios-utilities/blob/main/on-boot-script/README.md#install
My understanding is , running 
`curl -fsL "https://raw.githubusercontent.com/unifi-utilities/unifios-utilities/HEAD/on-boot-script/remote_install.sh" | /bin/sh `
on a UDM console will install this script and will also install CNI Plugins & CNI Bridge scripts.
As per Podman Update page it mentions that (https://github.com/unifi-utilities/unifios-utilities/tree/main/podman-update#requirements) 
Requirements
You have successfully set up the on boot script described here
05-install-cni-plugins.sh installed

I tried running the following command in unifi console and it shows the output. 

`:podman version`

```
Version:            1.6.1
RemoteAPI Version:  1
Go Version:         go1.12.10
OS/Arch:            linux/arm64
```

I am lost for the next part.
Can some one kindly explain the steps as per a linux user with very basic knowledge? 

Also a question: If I go wrong any where while doing this will it impact UDM router capability or may brick it?

**Comment by boostchicken** [comment]
Well, newer versions of Podman may not be compatible with your current firmware version, please look at the readme for links to current podman builds or the podman-update folder for instructions
